packages/starlight/utils/navigation.ts

```diff
@@ -64,7 +66,7 @@ function groupFromAutogenerateConfig(
   routes: Route[],
   currentPathname: string
 ): Group {
-  const { directory } = item.autogenerate;
+  const { collapsed: collapsedGeneratedEntries, directory } = item.autogenerate;
```

Comment:
This is totally a tiny detail, so please excuse me being fussy! I can understand renaming here for clarity, but I’d suggest this naming as it’s a bit more compact (collapsed!) and I think reads nicely alongside `item.collapsed`.
```suggestion
  const { collapsed: subgroupCollapsed, directory } = item.autogenerate;
```

Reply:
No worries, on the contrary, I even knew this was not a good name after hitting the refactor shortcut at least 4 times for this one ^^